orchagent/neighorch.cpp

```diff
@@ -202,23 +222,22 @@ bool NeighOrch::addNextHop(const IpAddress &ipAddress, const string &alias)
     sai_status_t status = sai_next_hop_api->create_next_hop(&next_hop_id, gSwitchId, (uint32_t)next_hop_attrs.size(), next_hop_attrs.data());
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to create next hop %s on %s, rv:%d",
-                       ipAddress.to_string().c_str(), alias.c_str(), status);
+        SWSS_LOG_ERROR("Failed to create next hop %s, rv:%d",
+                       nexthop.to_string().c_str(), status);
         return handleSaiCreateStatus(SAI_API_NEXT_HOP, status);
     }
 
-    SWSS_LOG_NOTICE("Created next hop %s on %s",
-                    ipAddress.to_string().c_str(), alias.c_str());
+    SWSS_LOG_NOTICE("Created next hop %s", nexthop.to_string().c_str());
 
     NextHopEntry next_hop_entry;
     next_hop_entry.next_hop_id = next_hop_id;
     next_hop_entry.ref_count = 0;
     next_hop_entry.nh_flags = 0;
     m_syncdNextHops[nexthop] = next_hop_entry;
 
-    m_intfsOrch->increaseRouterIntfsRefCount(alias);
+    m_intfsOrch->increaseRouterIntfsRefCount(nexthop.alias);
 
-    if (ipAddress.isV4())
+    if (!nexthop.label_stack.getSize())
```

Comment:
So if label_stack is present, you intend to increment CRM_IPV6_NEXTHOP?

Reply:
Merge issue.  This is fixed.